Change Form Template for the NFIQ 2 Dataset

elmiomar opened the pull request:
Updated the `datasets.yaml` config file to use the `no-disclaimer` form template for the NFIQ2 dataset.

RayPlante reviewed (approved):
The agreement question, "I have read and understood the disclaimer set out above", still appears, even though the disclaimer itself does.  Is this a hard-coded question?

elmiomar commented:
The disclaimer checkbox was missing Ngif directive. I added it and added a unit to verify that the disclaimer checkbox is conditionally hidden based on the presence of disclaimers in your **selectedFormTemplate** object.

RayPlante reviewed (approved):
I tested this combined with PRs #52 and [oar-lib-angular # 27](https://github.com/usnistgov/oar-lib-angular/pull/27), along with the associated updates to the distribution service (e.g. see [oar-dist-service PR# 102](https://github.com/usnistgov/oar-dist-service/pull/102)) all on testdata.  I confirmed that the disclaimer and associated question were removed from the request form for mds2-2909, that the request and approval apps continue to operate properly, and that the data could be downloaded.
